spec/routes.spec.ts

```diff
@@ -302,6 +302,28 @@ const routeTests = ({
       ]);
     });
 
+    it('supports strict zod objects', () => {
+      const routeParameters = generateParamsForRoute({
+        request: {
+          query: z.strictObject({
```

Comment:
Before this change this throws a type error.

Reply:
Nice! Thank you!